lib/frameworks/utils.rb

```diff
@@ -107,7 +107,7 @@ def save_and_link_screenshot
       embed "./reports/Screenshot_#{current_time}.png", 'image/png', "Actual screenshot of the error at #{current_url}"
     end
 
-    def parent_node(element,level_up)
+    def parent_node(element, level_up)
```

Comment:
Good to provide comment above this method, as no one will directly understand what this method does?